package.json

```diff
@@ -64,15 +63,12 @@
     "eslint-plugin-react": "^7.11.1",
     "exec-sh": "^0.3.0",
     "jest": "^23.6.0",
-    "lodash": "^4.17.11",
```

Comment:
https://github.com/plotly/dash-core-components/blob/6fac9549e6c1976f2e5ac406d8b8338d04b5193b/test/unit/Interval.test.js#L4

Kinda silly to bring in lodash just for that... replace with the `ramda` version?

But begs the question - why didn't the test fail?

Reply:
Perhaps it still builds due to https://github.com/plotly/dash-core-components/blob/6fac9549e6c1976f2e5ac406d8b8338d04b5193b/package-lock.json#L7417 in `package-lock.json`? It would be installed even though it is not an explicit dependency(?)

Reply:
ah good call @mjclawar, it's a sub-dep all over the place...
```
dash-core-components@0.43.0 /Users/alex/plotly/dash-core-components
├─┬ @babel/core@7.1.6
│ ├─┬ @babel/generator@7.1.6
│ │ └── lodash@4.17.11  deduped
│ ├─┬ @babel/traverse@7.1.6
│ │ └── lodash@4.17.11  deduped
│ ├─┬ @babel/types@7.1.6
│ │ └── lodash@4.17.11  deduped
│ └── lodash@4.17.11  deduped
├─┬ @babel/preset-env@7.1.6
│ ├─┬ @babel/plugin-proposal-unicode-property-regex@7.0.0
│ │ └─┬ @babel/helper-regex@7.0.0
│ │   └── lodash@4.17.11  deduped
│ ├─┬ @babel/plugin-transform-block-scoping@7.1.5
│ │ └── lodash@4.17.11  deduped
│ ├─┬ @babel/plugin-transform-classes@7.1.0
│ │ └─┬ @babel/helper-define-map@7.1.0
│ │   └── lodash@4.17.11  deduped
│ └─┬ @babel/plugin-transform-modules-amd@7.1.0
│   └─┬ @babel/helper-module-transforms@7.1.0
│     └── lodash@4.17.11  deduped
├─┬ babel-jest@23.6.0
│ └─┬ babel-plugin-istanbul@4.1.6
│   └─┬ istanbul-lib-instrument@1.10.2
│     ├─┬ babel-generator@6.26.1
│     │ └── lodash@4.17.11  deduped
│     ├─┬ babel-template@6.26.0
│     │ └── lodash@4.17.11  deduped
│     └─┬ babel-traverse@6.26.0
│       └── lodash@4.17.11  deduped
├─┬ css-loader@1.0.1
│ └── lodash@4.17.11  deduped
├─┬ enzyme@3.7.0
│ └─┬ cheerio@1.0.0-rc.2
│   └── lodash@4.17.11  deduped
├─┬ eslint@5.9.0
│ ├─┬ inquirer@6.2.0
│ │ └── lodash@4.17.11  deduped
│ ├── lodash@4.17.11  deduped
│ └─┬ table@5.1.0
│   └── lodash@4.17.11  deduped
├─┬ eslint-plugin-import@2.14.0
│ └── lodash@4.17.11  deduped
├─┬ jest@23.6.0
│ └─┬ jest-cli@23.6.0
│   ├─┬ jest-config@23.6.0
│   │ └─┬ babel-core@6.26.3
│   │   ├─┬ babel-register@6.26.0
│   │   │ ├─┬ babel-core@6.26.3
│   │   │ │ └── lodash@4.17.11  deduped
│   │   │ └── lodash@4.17.11  deduped
│   │   └── lodash@4.17.11  deduped
│   ├─┬ jest-environment-jsdom@23.4.0
│   │ └─┬ jsdom@11.12.0
│   │   └─┬ request-promise-native@1.0.5
│   │     └─┬ request-promise-core@1.1.1
│   │       └── lodash@4.17.11  deduped
│   └─┬ jest-runtime@23.6.0
│     └─┬ babel-core@6.26.3
│       └── lodash@4.17.11  deduped
├── lodash@4.17.11 
├─┬ react-dates@12.7.1
│ └── lodash@4.17.11  deduped
├─┬ react-docgen@2.21.0
│ └─┬ async@2.6.1
│   └── lodash@4.17.11  deduped
└─┬ styled-jsx@3.1.2
  └─┬ babel-types@6.26.0
    └── lodash@4.17.11  deduped
```

@Marc-Andre-Rivet let's remove this explicit ref anyway, then we'll be good to go.

Reply:
Good question.. or rather, why did the build pass at all..

Reply:
I think lodash is still being pulled by other libraries as a `requires` -- causing it to be present in node_modules, for example `@babel/core` pulls it in.

Reply:
Thanks for catching this

Reply:
Ah! I missed the answers on your side :)